French.json

```diff
@@ -0,0 +1,134 @@
+    "command.tphere.declined.receiver": "Vous avez refusé le requête de téléportation sur lui de {%sender}.",
+    "command.tphere.declined.sender": "{%receiver} a refusé votre demande de téléportation sur vous.",
+    "command.tphere.noperm": "Vous n'avez pas la permission d'émettre un requête de téléportation vers vous. Achetez les rangs §l§aEMERALD§r §cou §l§bLEGEND§r §§msur §§abngmc.co/store §§m!",
+    "command.tphere.norequest": "Ce joueur n'a pas envoyé de requêtes de téléportation vous étant destiné.",
+    "command.tphere.receive": "{%sender}  vous a envoyé une requête de téléportation sur lui. Utilisez §§a/tphere accept {%sender} pour l'accepté.",
+    "command.tphere.send": "Votre requête de téléportation sur vous à {%receiver} a été envoyé.",
+    "command.tpto.accepted.receiver": "Vous avez accepté la requête de téléportation sur vous de {%sender}.",
+    "command.tpto.accepted.sender": "{%receiver} a accepté votre demande de téléportation sur lui.",
+    "command.tpto.declined.receiver": "Vous avez refusé la requête de téléportation sur vous de {%sender}.",
+    "command.tpto.declined.sender": "{%receiver} a refusé votre requête de téléportation sur lui.",
+    "command.tpto.noperm": "Vous n'avez pas la permission d'émettre une requête de téléportation vers des joueurs. Achetez les rangs §l§aEMERALD§r §§mou §l§bLEGEND§r §§m sur §§angmc.co/store §§m!",
+    "command.tpto.norequest": "Ce joueur n'a pas envoyé de requêtes de téléportation vous étant destiné.",
+    "command.tpto.receive": "{%sender} vous a envoyé un requête de téléportation sur vous. Utilisez §§a/tpto accept {%0} pour l'accepter.",
+    "command.tpto.send": "Votre requête de téléportation sur {%receiver} a été envoyé.",
+    "command.vanish.disabled": "Vous êtes maintenant visible!",
+    "command.vanish.enabled": "Vous êtes maintenant invisible!",
+    "command.social.nosupport": "Ce serveur ne supporte pas les caractéristiques sociales.",
+    "command.social.notfound": "Désolé, ce joueur n'est pas trouvé.",
+    "command.vote.slowdown": "Ralentissez, nous sommes en train de vérifier votre vote.",
+    "db.error": "Une erreur de database inattendu a eu lieu lors de votre connexion dessus. Veuillez ré-essayer plus tard.",
+    "formhandler.reporter.notreported": "Ce joueur ne peut être signalé.",
+    "formhandler.reporter.reported": "Vous avez signalé {%reported} pour {%reason}.",
+    "formhandler.reporter.staffmessage": "{%reported} a été signalé par {%reporter} pour {%reason}.",
+    "formhandler.settings.disabled": "Désactiver",
+    "formhandler.settings.enabled": "Activer",
+    "formhandler.settings.fly": "{%enabledOrDisabled} voler dans le lobby.",
+    "formhandler.settings.fly.hidingrank": "Vous ne pouvez pas voler pendant que vous utilisez un surnom ou cachez votre rang.",
+    "formhandler.settings.friends": "Vous êtes maintenant {%enabledOrDisabled} demandes d'ami.",
```

Comment:
Is `{%enabledOrDisabled}` will be replaced with the translation of `enabled` or `disabled`?